db/schema.rb

```diff
@@ -653,8 +653,8 @@
   create_table "status_pins", force: :cascade do |t|
     t.bigint "account_id", null: false
     t.bigint "status_id", null: false
-    t.datetime "created_at", default: -> { "now()" }, null: false
-    t.datetime "updated_at", default: -> { "now()" }, null: false
+    t.datetime "created_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
```

Comment:
What's going on here?

Reply:
Seems that `CURRENT_TIMESTAMP` was added to Rails 5. I'm not sure why it decided to change in this PR(and I can change it back if needed). This seems like the more rails-y way to do it, as it is database-agnostic.

Reply:
Yes, I'd like to minimize the changes as much as possible. We don't need to be database-agnostic anyway because we use postgres-specific query features.

Reply:
This should be fixed now